Assets/MRTK/Core/Utilities/Editor/AssemblyDefinition.cs

```diff
@@ -223,6 +225,38 @@ public void Save(string fileName)
                 file.IsReadOnly = true;
             }
         }
+
+        /// <summary>
+        /// Adds a reference to an existing assembly definition.  Make sure to call Save() after using this method to save the changes.
+        /// </summary>
+        /// <param name="referenceName">The name of the reference to add to this assembly definition</param>
+        public void AddReference(string referenceName)
```

Comment:
Nice! Doesn't have to be in this PR if we want to keep the scope to Leap, but should we update the rest of the configuration checkers to use this new method?

Reply:
Yes we could, also in the future I could add an option for a list of reference names instead of just the one to make it easier